stix_shifter_modules/elastic_ecs/tests/stix_translation/test_elastic_ecs_json_to_stix.py

```diff
@@ -848,4 +871,13 @@ def test_observer(self):
           observer_ip and 
           observer_ip.get("type") == "ipv4-addr" and
           observer_ip.get("value") == "10.0.0.101" 
-        )
\ No newline at end of file
+        )
+
+    def test_email(self):
+        result_bundle = run_in_thread(entry_point.translate_results, data_source, [email_data])
+        assert (result_bundle['type'] == 'bundle')
+        translation_objects = result_bundle.get('objects')
+        assert (translation_objects and len(translation_objects) == 2)
+        observed_data = translation_objects[1]
+        stix_objects = observed_data.get("objects")
+        assert len(stix_objects) == 8
```

Comment:
i would put more specific test to verify if the objects are created correctly.

Reply:
More specific tests added.